module/VuFindSearch/src/VuFindSearch/Backend/Primo/Backend.php

```diff
@@ -246,10 +246,13 @@ protected function paramBagToPrimoQuery(ParamBag $params)
             $options[$key] = in_array($key, $arraySettings) ? $param : $param[0];
         }
 
-        // Use special facet pcAvailabilty if it has been set
-        if (isset($params['filterList']['pcAvailability'])) {
+        // Use special pcAvailability filter if it has been set:
+        if ($values = $params['filterList']['pcAvailability']['values'] ?? []) {
+            $value = $values[0];
```

Comment:
If `$params['filterList']['pcAvailability']['values']` is unset and you default to `[]`, then I think that `$values[0]` will be unset and cause a notice. I guess the unset case is not covered by the tests below!

Reply:
It won't go into the `if` branch since [] is falsy, right?

Reply:
Anyway, I replaced `$values[0]` with `reset($values)` for extra safety.

Reply:
You are, of course, correct -- I wasn't thinking it all the way through. In any case, thanks for the adjustment!